lib/platform/github/gh-got-wrapper.js

```diff
@@ -19,7 +33,11 @@ async function get(path, options, retries = 5) {
   opts.baseUrl = opts.endpoint;
   delete opts.endpoint;
   const method = opts.method || 'get';
-  const useCache = opts.useCache || true;
+  const useCache = opts.useCache || true; // in-memory cache
+  opts.cache = cacheMap; /* persistent cache - always validated, but
+                            hits don't count against request limit */
+  opts.shared = false; // https://github.com/kornelski/http-cache-semantics/issues/9
```

Comment:
Does this mean that caching is done uniquely based on the authentication token used, for example?

Reply:
My understanding of this is that RFC 7234 forbids caching in this situation as an additional precaution against serving private cached data to unauthenticated requests. However, servers should already specify the header used for authentication in the `Vary` response header (at least GitHub does), and, assuming the cache driver implements this mechanism correctly, this "should" work as intended.